metrics/grafana/tikv_details.json

```diff
@@ -8303,7 +8303,7 @@
             "x": 12,
             "y": 38
           },
-          "id": 4375,
+          "id": 91991299,
```

Comment:
Is this change unintentional?

Reply:
yes, maybe I accidentally brought in when I upgraded grafana a few days ago